docs/patterns/messaging-bridge-content.md

```diff
@@ -0,0 +1,88 @@
+The Messaging Bridge design pattern is a way to integrate disparate systems built on top of different messaging infrastructures.
+
+## Context and problem
+
+Many organizations end up in a situation in which their portfolio of IT systems uses multiple messaging infrastructures, e.g. by way of mergers and acquisitions. Others try to extend their current on-premises systems with cloud-hosted components for ease of maintenance, cost-effectiveness or other reasons. Yet others find themselves in a position where they need to integrate their newer systems with legacy ones. 
+
+One common solution to these challenges is dedicated integration through HTTP-based web services. In that approach systems being integrated are modified to include HTTP endpoints and/or clients in order to exchange the data.
+
+But this has a number of drawbacks:
+- it requires the applications being integrated to be modified (by adding an HTTP client on one side and an HTTP request handler on the other)
+- it requires hosting of HTTP endpoints, adding the complexity related to making these services secure and highly-available
+- it is prone to network connectivity problems, requiring custom-build retry mechanisms
+
+## Solution
+
+If the systems being integrated are themselves message-based (i.e. consisting of components that communicate by exchanging messages), a solution that solves the integration problem while mitigating the drawbacks listed above is the Messaging Bridge pattern.
+
+In that case each system already connects to one messaging infrastructure. In order to integrate them, a bridge component is introduced that is able to connect to two (or more) messaging infrastructures at the same time, pulling messages from one and pushing them to the other without changing the payload.
+
+Neither of the systems being integrated needs to know about the other, nor the bridge. The sender system is configured to send specific messages to a designated queue. The messages are picked up by the bridge and fowarded to another queue in a different messaging infrastructure where they are finally picked up by the receiver system.
+
+### Benefits
+
+- The systems integrated via the Messaging Bridge do not have to be modified; ideally, the endpoints would not be aware that the messages are bridged.
+- The message-driven integration is generally more reliable, compared to the HTTP alternative due to the at-least-once message delivery mechanism.
```

Comment:
```suggestion
- The integration is generally more reliable, compared to the HTTP alternative due to the at-least-once message delivery mechanism guarantee.
- Migration scenarios can be more flexible. For example, endpoints can be migrated from one messaging infrastructure to another as the schedule permits instead of all-at-once.
```